install/ui/src/freeipa/policy.js

```diff
@@ -132,4 +135,4 @@ exp.register = function() {
 phases.on('registration', exp.register);
 
 return exp;
-});
\ No newline at end of file
+});
```

Comment:
Please, turn off newlines at the end of file in your text editor and/or fix this. Other than that LGTM.